### PR TITLE
fix: remove generated files from examples index

### PR DESCRIPTION
**Pull request by jkaster**
Added more `build` folders to ignore for the code miners. API Explorer was showing links to files that were build artifacts and don't get checked in to the repository, so they were dead links on the developer portal.

### CODE REVIEW

File: packages/sdk-codegen-scripts/src/declarationMiner.ts [hunk 1]
```diff
@@ -127,8 +128,11 @@ export class DeclarationMiner {
   }
 
   get remoteOrigin(): string {
+    if (this.originOverride) {
+      return this.originOverride
+    }
     process.chdir(this.sourcePath)
-    return getRemoteHttpOrigin()
+    return getRemoteOrigin()
```
Comment (bryans99) [inline packages/sdk-codegen-scripts/src/declarationMiner.ts:135]: Is this what we really want. I looked at the code and it executes
`git remote get-url origin`
which returns
`git@github.com:looker-open-source/sdk-codegen.git`
Looke like getRemoteHttpOrigin attemps to convert it

File: packages/sdk-codegen-scripts/src/exampleMiner.ts [hunk 2]
```diff
@@ -80,7 +80,17 @@ export const filterCodeFiles = (fileName: string) => {
...
 
-const skipFolder = (name: string, excludeList: string[] = ['node_modules']) =>
+export const IGNORE_PATHS = [
+  'node_modules',
+  'lib',
+  'dist',
+  'bazel-bin',
+  'build',
+  'bin',
+  '.build',
```
Comment (bryans99) [inline packages/sdk-codegen-scripts/src/exampleMiner.ts:90]: .direnv, .github, .vscode, .idea, results?

Probably overkill

**Review by bryans99 — APPROVED** [review]
LGTM with a couple of questions

File: packages/sdk-codegen-scripts/src/declarationMiner.ts (hunk 1 above)
Comment (jkaster) [reply packages/sdk-codegen-scripts/src/declarationMiner.ts:135]: Yes. Note this is declaration miner, not example miner. The address for the helltool code repo is no longer https when you ask for the remote origin. That's why I added the `origin_override` configuration and alternate path composition.

File: packages/sdk-codegen-scripts/src/exampleMiner.ts (hunk 2 above)
Comment (jkaster) [reply packages/sdk-codegen-scripts/src/exampleMiner.ts:90]: Easy enough to add. I'll do that now.

Comment (jkaster) [reply packages/sdk-codegen-scripts/src/exampleMiner.ts:90]: Added

**Review by bryans99 — APPROVED** [review]

**Pull request merged**